scipy/stats/contingency.py

```diff
@@ -271,3 +272,122 @@ def chi2_contingency(observed, correction=True, lambda_=None):
                                    lambda_=lambda_)
 
     return chi2, p, dof, expected
+
+def association(arr, chi2_stat, stat="cramer"):
```

Comment:
Add another blank line before this `def` (see the PEP-8 guidelines about [blank lines](https://www.python.org/dev/peps/pep-0008/#blank-lines)).
```suggestion

def association(arr, chi2_stat, stat="cramer"):
```

Reply:
done.